### DISCUSSION TITLE
How to use dark mode?

### DISCUSSION THREAD

**Discussion by nelsonlaidev** [heroui-inc/heroui#573] [Help] [answered]
Sorry, I already read the documentation, but I have some problems.
- Do I need to duplicate the config such as colors, fonts in both themes?
- My custom font is not working in dark mode (it only works in light mode).
- My custom breakpoints are not working in both themes.
- Most of the time I need to restart the server to apply my change in theme config(sometimes it fails even if I delete the `.next` folder).
- Why there is some empty property in CSS?

Reproduce: [StackBlitz](https://stackblitz.com/edit/nextjs-xfgwyv?file=lib/theme.ts)

Restart server problem:

https://user-images.githubusercontent.com/75498339/175805530-6f8106d2-3579-4c2a-814a-6975a3741255.mp4

Empty property in CSS:
![Snipaste_2022-06-26_16-23-45](https://user-images.githubusercontent.com/75498339/175806059-d75d8398-a569-4199-9266-9e95afe0cacd.png)

**Comment by cyanxiao** [comment, marked as answer]
If you want use a customized light/dark theme, you may need to duplicate the config, otherwise you'll use preset configs from default theme.

**Reply by nelsonlaidev** [reply]
Thanks, I'll look into it.
